src/ui/ducks/identities.ts

```diff
@@ -72,13 +73,21 @@ export const createIdentityRequest = () => async (): Promise<void> => {
   await postMessage({ method: RPCAction.CREATE_IDENTITY_REQ });
 };
 
+interface ICreateIdentityArgs {
```

Comment:
Related to #264, we can move all internal ducks types to one main internal type script in ducks folder. `src/ui/ducks/types.ts`